src/libraries/Common/src/Interop/SunOS/procfs/Interop.ProcFsStat.TryReadProcessStatusInfo.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class procfs
+    {
+        /// <summary>
+        /// Attempts to get status info for the specified process ID.
+        /// </summary>
+        /// <param name="pid">PID of the process to read status info for.</param>
+        /// <param name="processStatus">The pointer to processStatus instance.</param>
+        /// <returns>
+        /// true if the process status was read; otherwise, false.
+        /// </returns>
+        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_ReadProcessStatusInfo", SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
```

Comment:
This return attribute isn't needed.

Reply:
Will remove. There are few other instances on Unix with similar pattern, e.g.: https://github.com/dotnet/runtime/blob/b94c6458459fed86a866ed73e2c30ae466c47687/src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ASN1.cs#L50-L52